packages/sdk/src/predefinedGeneratorResolvers.ts

```diff
@@ -75,6 +75,8 @@ export const predefinedGeneratorResolvers: PredefinedGeneratorResolvers = {
     await checkTypeScriptVersion()
 
     if (!prismaClientDir && !process.env.PRISMA_GENERATE_SKIP_AUTOINSTALL) {
+      // TODO: `prisma generate` may be called deeper than one subdirectory from
+      // the package root.
```

Comment:
What is the potential impact of this?

Reply:
The potential impact is creating a phony `package.json` file somewhere in the project. Shouldn't actually break anything if that happens, but could be annoying.

Reply:
Ok I see. Yeah, that is not optimal.